core/src/test/scala/io/projectglow/vcf/VCFDatasourceSuite.scala

```diff
@@ -85,7 +85,6 @@ class VCFDatasourceSuite extends GlowBaseTest {
   }
 
   test("check parsed row") {
```

Comment:
I think we should have something like this for FastReader too where the assertion is against row with flattened info fields.

Reply:
I adapted the suite that checks flattened info fields to check the entire parsed row.